### PR TITLE
Additional guidelines for how to use Conda focusing on tasks

### PR DESCRIPTION
**Pull request by YankeMao**
Write the instructions for four major types of tasks of conda, which are Managing conda, Managing environments, Managing channels, and Managing packages. Introduced the guidelines of these four tasks and the commonly used command for these tasks.

### CODE REVIEW

File: Topics/Development_Process/Conda/Tasks_Conda.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+## Managing Conda
```
Comment (emma0925) [inline Topics/Development_Process/Conda/Tasks_Conda.md:1]: Maybe you can dd a general description in the beginning of the file

File: Topics/Development_Process/Conda/Tasks_Conda.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+### Suppressing Warning Messages About Conda Updates
+
+- Command to Suppress Warnings: `conda config --set notify_outdated_conda false`
+- This command stops Conda from displaying warnings about newer versions being available.
+- Alternatively, you can manually add `notify_outdated_conda: false` to your `.condarc` file.
+
+For more comprehensive guidance, refer to the official documentation on [Managing Conda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-conda.html).
+
+## Managing environments
+
```
Comment (emma0925) [inline Topics/Development_Process/Conda/Tasks_Conda.md:29]: Maybe you can also add a description of what the section does for each sub-sections

**Review by emma0925 — COMMENTED** [review]
Hi Yanke, I think your pr looks great. Here are a few suggestions for making it more clear.
1. Add a  few sentences on the high level idea of managing conda in the beginning of the file 
2. You can also consider adding a description for each sub-section like managing environments
3. Also, I think it would be very helpful to add a table of contents in the begining

File: Topics/Development_Process/Conda/Tasks_Conda.md (hunk 2 above)
Comment (YankeMao) [reply Topics/Development_Process/Conda/Tasks_Conda.md]: Added high-level description for each type of tasks.

File: Topics/Development_Process/Conda/Tasks_Conda.md (hunk 1 above)
Comment (YankeMao) [reply Topics/Development_Process/Conda/Tasks_Conda.md]: Thank you for your suggestion! I splited the file into four separate files and each file introduced one kind of task. And I add a high-level description for each file at the beginning.

**Comment by YankeMao** [comment]
> Hi Yanke, I think your pr looks great. Here are a few suggestions for making it more clear.
> 
> 1. Add a  few sentences on the high level idea of managing conda in the beginning of the file
> 2. You can also consider adding a description for each sub-section like managing environments
> 3. Also, I think it would be very helpful to add a table of contents in the begining

File: Topics/Development_Process/Conda/Managing_channels.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+
+## Managing Channels
```
Comment (emma0925) [inline Topics/Development_Process/Conda/Managing_channels.md:2]: Maybe you can change it to a bigger font using only one '#', so it consists with pull request #600

File: Topics/Development_Process/Conda/Managing_channels.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Information about channels and how they are searched through when installing packages.
+
+### Conda Channels
```
Comment (emma0925) [inline Topics/Development_Process/Conda/Managing_channels.md:6]: "##" for here

File: Topics/Development_Process/Conda/Managing_environments.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
+## Managing environments
```
Comment (emma0925) [inline Topics/Development_Process/Conda/Managing_environments.md:1]: also only one '#' here, same thing for the rest of the document

Comment (YankeMao) [reply Topics/Development_Process/Conda/Managing_environments.md]: Edited.

File: Topics/Development_Process/Conda/Managing_channels.md (hunk 4 above)
Comment (YankeMao) [reply Topics/Development_Process/Conda/Managing_channels.md]: Edited.

File: Topics/Development_Process/Conda/Managing_channels.md (hunk 3 above)
Comment (YankeMao) [reply Topics/Development_Process/Conda/Managing_channels.md]: Edited.

File: Topics/Development_Process/Conda/Manging_Conda.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- This command will check for the latest version of Conda and prompt you to update ('y' to proceed with the update). It also informs about other packages that will be updated or changed as a result of the Conda update.
+
+## Suppressing Warning Messages About Conda Updates
+
+- **Command to Suppress Warnings**: `conda config --set notify_outdated_conda false`
+- This command stops Conda from displaying warnings about newer versions being available.
+- Alternatively, you can manually add `notify_outdated_conda: false` to your `.condarc` file.
+
+For more comprehensive guidance, refer to the official documentation on [Managing Conda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-conda.html).
+
```
Comment (emma0925) [inline Topics/Development_Process/Conda/Manging_Conda.md:27]: I think you should add a reference section in the end

Comment (YankeMao) [reply Topics/Development_Process/Conda/Manging_Conda.md:27]: Thanks for the reminding! Just added

**Review by emma0925 — APPROVED** [review]
LGTM

**Pull request merged**